core/tests/tests/Number.mint

```diff
@@ -54,6 +54,17 @@ suite "Number.fromString" {
     |> Maybe.isNothing()
   }
 
+  test "returns nothing if it does not get passed a string" {
+    Maybe.oneOf(
+      [
+        Number.fromString(`true`),
+        Number.fromString(`0`),
+        Number.fromString(`{x: 0}`),
+        Number.fromString(`/x/`)
+      ])
+    |> Maybe.isNothing()
+  }
+
```

Comment:
I don't think we ought to test the abuse of the language via backticks usage.

Reply:
Yes, but as it was explicitly requested, we should also add tests for it in this case, in my opinion.